content/en/docs/contribute/participating.md

```diff
@@ -75,6 +75,56 @@ the pull request from being merged. Any member can also remove a `/hold` comment
 to cause a PR to be merged if it already has both `/lgtm` and `/approve` applied
 by appropriate people.
 
+#### Becoming a member
+
+After you have successfully submitted at least 5 substantive pull requests, you
+can request [membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member)
+in the Kubernetes organization. Follow these steps:
+
+1.  Find two reviewers or approvers to [sponsor](/docs/contribute/advanced#sponsor-a-new-contributor)
+    your membership.
+    
+      Ask for sponsorship in the #sig-docs channel on the 
+      Kubernetes Slack instance](https://kubernetes.slack.com) or on the
+      [SIG Docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
+      
+      {{< note >}}
+      Don't send a direct email or Slack direct message to an individual
+      SIG Docs member.
+      {{< /note >}}
+
+2.  Send an email to the [Kubernetes membership request list](mailto:kubernetes-membership@googlegroups.com)
+    and add your two sponsors and any other relevant people to the CC of the
+    email. Use the following template.
+    
+      ```plaintext
+        I have joined kubernetes-dev@googlegroups.com and fulfilled all the
```

Comment:
The extra level of indentation here is necessary because of a Hugo bug. It adds extra spaces in front of the lines in the output, but it's relatively benign.